Cannot link for x86_64 when compiling (OSX)

Hi there, I've been trying to compile aqemu. Everything seems to be working well, but after installing the dependencies and typing `make` the build didn't finish, unfortunately.

I'm on OSX and installed my dependencies through brew. Libvncserver isn't on brew at the moment, but I found a [tap](https://github.com/msikma/homebrew-local/) for it.

My cmake output:

```
-- Performing Test LIBVNCSERVER_FOUND
-- Performing Test LIBVNCSERVER_FOUND - Success
-- Found LibVNCServer: /usr/local/lib/libvncserver.dylib
-- MAN page will be installed is not packed.
-- MAN_PAGE_COMPRESSOR flag not set. Available values: gzip, bzip2, lzma
-- Will be installed not compressed MAN pages
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/msikma/Downloads/aqemu-0.9.2
```

`make` output:

```
[  3%] Built target icons
[  5%] Automatic MOC and UIC for target aqemu
[  5%] Built target aqemu_autogen
[  6%] Linking CXX executable aqemu
Undefined symbols for architecture x86_64:
  "System_Info::Update_Host_USB()", referenced from:
      Add_USB_Port_Window::on_Button_Update_Host_USB_clicked() in Add_USB_Port_Window.cpp.o
      System_Info::Get_All_Host_USB() in System_Info.cpp.o
      Virtual_Machine::Build_QEMU_Args() in VM.cpp.o
  "System_Info::Get_Host_FDD_List()", referenced from:
      Emulator_Control_Window::Create_Device_Menu() in Emulator_Control_Window.cpp.o
      Main_Window::Update_Recent_Floppy_Images_List() in Main_Window.cpp.o
      Properties_Window::Set_Floppy(VM_Storage_Device const&, QString const&) in Properties_Window.cpp.o
  "System_Info::Get_Host_CDROM_List()", referenced from:
      Emulator_Control_Window::Create_Device_Menu() in Emulator_Control_Window.cpp.o
      Main_Window::Update_Recent_CD_ROM_Images_List() in Main_Window.cpp.o
      Properties_Window::Set_CD_ROM(VM_Storage_Device const&, QString const&) in Properties_Window.cpp.o
  "System_Info::Get_Free_Memory_Size(int&, int&)", referenced from:
      Main_Window::on_TB_Update_Available_RAM_Size_clicked() in Main_Window.cpp.o
      Main_Window::on_CH_Remove_RAM_Size_Limitation_stateChanged(int) in Main_Window.cpp.o
      VM_Wizard_Window::on_CH_Remove_RAM_Size_Limitation_stateChanged(int) in VM_Wizard_Window.cpp.o
      VM_Wizard_Window::on_TB_Update_Available_RAM_Size_clicked() in VM_Wizard_Window.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [aqemu] Error 1
make[1]: *** [CMakeFiles/aqemu.dir/all] Error 2
make: *** [all] Error 2
```

Unfortunately I'm not really sure how to begin debugging this. I assume that one of my dependencies (or maybe aqemu itself?) is being built for 32 bit instead of 64 bit. I've not been able to find a solution through Google so far.

Thanks for any help you might be able to offer!

* uname -a: `Darwin Fuji.local 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64 x86_64`
* OSX 10.13.4, command line tools 10.1
* aqemu v0.9.2

--- a/src/System_Info.cpp
+++ b/src/System_Info.cpp
@@ -2732,6 +2732,93 @@ bool System_Info::Update_Host_USB()
 
 #endif // FreeBSD
 
+#ifdef Q_OS_MAC
+
+#include <sys/sysctl.h>
+#include <mach/host_info.h>
+#include <mach/mach_host.h>
+#include <mach/task_info.h>
+#include <mach/task.h>
+#include <QDir>
+#include <QFileInfoList>
+
+void System_Info::Get_Free_Memory_Size( int &allRAM, int &freeRAM )
+{
+	int mib[6]; 
+	mib[0] = CTL_HW;
+	mib[1] = HW_PAGESIZE;
+
+	int page_size;
+	size_t length;
+	length = sizeof (page_size);
+	if (sysctl (mib, 2, &page_size, &length, NULL, 0) < 0)
+	{
+				AQError( "void  System_Info::Get_Free_Memory_Size( int &allRAM, int &freeRAM )",
+				 "Cannot Get Memory Pagesize! (sysctl)" );
+	}
+
+	mach_msg_type_number_t count = HOST_VM_INFO_COUNT;
+
+	vm_statistics_data_t vmstat;
+	if (host_statistics (mach_host_self (), HOST_VM_INFO, (host_info_t) &vmstat, &count) != KERN_SUCCESS)
+	{
+		AQError( "void  System_Info::Get_Free_Memory_Size( int &allRAM, int &freeRAM )",
+				 "Cannot Get Information on Memory! (host_statistics)" );
+	}
+	
+	double all_ram = vmstat.wire_count + vmstat.active_count + vmstat.inactive_count + vmstat.free_count;
+	double free_ram = vmstat.free_count;
+	double cached_ram = vmstat.inactive_count;
+		
+	all_ram *= page_size;
+	free_ram *= page_size;
+	cached_ram *= page_size;
+	
+	allRAM = (int)(all_ram / 1024.0 / 1024.0);
+	freeRAM = (int)((free_ram + cached_ram) / 1024.0 / 1024.0);
+}
+
+QStringList System_Info::Get_Host_FDD_List()
+{
+	// Find Floppy's
+	QDir dev_dir( "/dev/fd/" );
+	QFileInfoList devices = dev_dir.entryInfoList( QStringList("*"), QDir::System, QDir::Name );
+	
+	QStringList tmp_list;
+	
+	for( int d = 0; d < devices.count(); ++d )
+	{
+		tmp_list << devices[d].absoluteFilePath();
+	}
+	
+	return tmp_list;
+}
+
+QStringList System_Info::Get_Host_CDROM_List()
+{
+	// Find CD-ROM's
+	QDir dev_dir( "/dev/" );
+	QFileInfoList devices = dev_dir.entryInfoList( QStringList("acd*"), QDir::System, QDir::Name );
+	
+	QStringList tmp_list;
+	
+	for( int d = 0; d < devices.count(); ++d )
+	{
+		tmp_list << devices[d].absoluteFilePath();
+	}
+	
+	return tmp_list;
+}
+
+bool System_Info::Update_Host_USB()
+{
+	AQError( "System_Info::Update_Host_USB()",
+			 "Not implemented!" );
+	return false;
+}
+
+#endif // macOS
+
 #ifdef Q_OS_WIN32
 
 #include <windows.h>

